eventmesh-connector-plugin/eventmesh-connector-rocketmq/src/main/java/org/apache/eventmesh/connector/rocketmq/consumer/RocketMQConsumerImpl.java

```diff
@@ -48,7 +51,7 @@
 import io.openmessaging.api.MessageSelector;
 import io.openmessaging.api.MessagingAccessPoint;
 
-public class RocketMQConsumerImpl implements MeshMQPushConsumer {
+public class RocketMQConsumerImpl implements Consumer {
```

Comment:
`MeshMQPushConsumer` is a service provider interface, if you remove `MeshMQPushConsumer` here, the `RocketMQConsumerImpl` can load success?

Reply:
`Consumer` has replace the `MeshMQPushConsumer`

Reply:
`MeshMQPushConsumer` is replaced by `Consumer`

Reply:
If so, it's needed to change the config file in META-INF/eventmesh `org.apache.eventmesh.api.consumer.MeshMQPushConsumer` to `org.apache.eventmesh.api.consumer. Consumer `

Reply:
yes, we need to test carefully before we merge to the `develop` branch

Reply:
Ok, this can be fixed later.